docs/azure_ad_config_guide.rst

```diff
@@ -138,3 +138,92 @@ Add this path to your projects ``urls.py`` file.
         path('oauth2/', include('django_auth_adfs.urls')),
         ...
     ] 
+    
+Step 3 - Register and configure an Azure AD frontend application
+--------------------------------------------------------
+Just like we did with our backend application in step 1 We have to register a new app for our frontend. Here we need to set it to SPA and the redirect URI value must match with the domain where your frontend application is located(eg. http://localhost:3000).
+
+
+
+
+.. image:: _static/2021/09_register_frontend_app.PNG
+    :scale: 50 %
+
+------------
+
+Copy your frontend's client ID, you will need later
+
+
+
+.. image:: _static/2021/10_copy-frontend-client_id.png
+    :scale: 50 %
+
+------------
+
+Now we need to add a scope of permissions to our API. 
+Navigate back to app registrations and click on your backend application.
+Go to **Expose an API** in the sidebar and press **add a scope**.
+
+
+.. image:: _static/2021/11-navigate_to_expose_an_api.PNG
+    :scale: 50 %
+
+------------
+
+If you have not created an Application ID URI, it will be autogenerated for you. Select it and press **save and continue**.
+
+
+.. image:: _static/2021/13_set_app_id.PNG
+    :scale: 50 %
+
+------------
+
+Then we will create the actual scope. Call it "read", and just fill in all the required fields with "read" (maybe write an actual description).
+
+
+
+.. image:: _static/2021/14_add_a_scope.PNG
+    :scale: 50 %
+
+------------
+
+Now we are going to add our frontend application as a trusted app for our backend. Press **add a client application**
+
+
+.. image:: _static/2021/15_add_authorized_app_1.png
+    :scale: 50 %
+
+------------
+
+Here you need to paste in your frontend application (client) id.
+
+
+.. image:: _static/2021/16_add_authorized_app_2.PNG
+    :scale: 50 %
+
+------------
+
+Now navigate back to app registrations. Click on your **frontend** application and navigate to API permissions. Press **add a permisson**.
+
+
+.. image:: _static/2021/17_navigate_to_api_permissions.PNG
+    :scale: 50 %
+
+------------
+
+Then we have to press **My API's** and then select the backend application. (This could be different if you don't have owner rights of the backend application.)
+
+
+.. image:: _static/2021/18_add_permission.PNG
+    :scale: 50 %
+
+------------k
+
+Here we can give our frontend the permission scope we created earlier. Press **Delegated permissions** (should be default) and select the permission you created and press **add permission**
+
+
+
+.. image:: _static/2021/19_add-permission-2.PNG
+    :scale: 50 %
+
+------------
```

Comment:
The docs builder will complain if you build with this at the end, so it has to be removed :scissors: